index.js

```diff
@@ -28,33 +28,41 @@ const Colors = {
  * @param {string} [config.moduleName = ''] - The name of the module to export
  * @param {RegExp[]} [config.internalImportPaths = []] - The internal import paths that will be parsed out
  */
-module.exports = function (config) {
-    return through.obj(function (vinylFile, encoding, callback) {
+module.exports = function(config) {
```

Comment:
Can you modify prettier to leave the space after the function?

Reply:
Why don't you use arrow functions?

Reply:
I can't leave a space after the function, Prettier is pretty opinionated. Those are the only options you can play with: https://prettier.io/docs/en/options.html

I'll switch for arrow functions.